SteamcLog/Classes/SteamcLog.swift

```diff
@@ -215,41 +215,48 @@ public struct SteamcLog {
 
     // MARK: Nonfatal Log Level
 
-    private func internalError(_ message: String, functionName: StaticString, fileName: StaticString, lineNumber: Int) {
+    private func internalError(_ message: StaticString, info: String? = nil, functionName: StaticString = #function, fileName: StaticString = #file, lineNumber: Int = #line) {
+        if let info = info {
+            xcgLogger.info(info, functionName: functionName, fileName: fileName, lineNumber: lineNumber)
```

Comment:
This approach is maybe good enough for now, but I don't love it. The issue here is that I think the regular filtering rules will apply to this, so if you are in "release" mode, this info will be lost. Ideally this would short circuit the regular filtering logic in some way so that it gets included as a breadcrumb even if `info` level filtering is turned off (since this is logically part of the error). Another alternative would be to actually include is as a separate field in the error object (like a new tag or something), but I don't think that would be any easier to implement

Reply:
Are we okay with the log functions at this level 'knowing' about Sentry? It seems a little iffy to me to have the log functions know about the log destinations.

Reply:
Yeah, that's the dilemma here, I don't think we DO want it to know about anything other then the normal xcglogger interface at this level (though we've got other versions of this problem coming, implementing https://github.com/steamclock/steamclog-swift/issues/50 is gonna have similar issues with needing to pass stuff down to the lower levels in different ways).

Two options that occur to me:
   - Repurpose some of the unused log levels to help with this, for example, we could grab one of the more severe log levels that we don't use at all, like `alert` and repurpose that as the carrier for these, then they make through the level filters (because they are worse than warnings), but the sentry destination would know "errors" get issues capture for them, but "alerts" only get breadcrumbs.
  - Instead of doing this as an info call, pass it down in the `userInfo` dictionary, and then the individual destinations have to know to pull it out and handle it appropriately (this is almost certainly how we'd have to do it for something like passing in an `Error` object).

Both of those have issues, which is why maybe we should just punt on this for now and go with the current implementation, but it is something it'd be nice to resolve eventually.

Reply:
Punting for now, created https://github.com/steamclock/steamclog-swift/issues/70 to come back to this later.